[#161469512] Promocoes

rafadescontinuado opened the pull request:
versão final + teste de aceitação

rafadescontinuado committed:
classe promocoes atributos ,construtor e metodos, criação da activit de  promoções

rafadescontinuado committed:
navegação activity de  promoções

rafadescontinuado committed:
versão beta tela de promoções

rafadescontinuado committed:
versão final tela de promoções ,imagens do storage

rafadescontinuado committed:
resolução bug rela inical

rafadescontinuado committed:
Teste de aceitação ,Resolução de busgs

DanilloMLS commented:
Existem imports não usados em DefaultTest.java, em PromocoesActivitySteps.java e em PerfilActivity.java
Há também campos não usados em PromocoesActivity.java e em PromocaoAdapter.java
Em PromocoesActivity.java e PromocaoAdapter.java sugere-se usar o modificador de acesso para privado em métodos e campos que serão usados apenas dentro da classe.

fdoliv commented on app/src/main/java/br/com/belapp/belapp/activities/ClienteLogadoActivity.java:
```diff
@@ -372,7 +372,9 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-
+            Intent intent = new Intent(ClienteLogadoActivity.this, PromocoesActivity.class);
+            startActivity(intent);
+            Toast.makeText(ClienteLogadoActivity.this, "Promoçõess", Toast.LENGTH_SHORT).show();
```
Utilize o arquivo `strings.xml` para declarar a string Promoções. Corrija o 's' a mais no fim da palavra.

fdoliv replied:
Você pode chamar o resource `R.string.title_activity_promocoes` ao invés de fazer uma nova declaração. 
Utilize `getText(R.string.title_activity_promocoes)` para acessar o recurso

fdoliv commented on app/src/main/java/br/com/belapp/belapp/activities/PromocoesActivity.java:
```diff
@@ -1,13 +1,35 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_promocoes);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
```
Adição de `Toolbar` com título para orientar em qual janela o usuário se encontra. Muito bom.

fdoliv commented on app/src/main/java/br/com/belapp/belapp/activities/PromocoesActivity.java:
```diff
@@ -1,28 +1,199 @@
+    private double longitude;
+    private DatabaseReference databaseReference;
+    private String curtida = "1";
+    String datausuario="";
```
Tente seguir os padrões de declaração de variáveis, disponível no [documento de boas práticas](https://github.com/FagnerPulca/Belapp/wiki/Boas-Pr%C3%A1ticas-de-Desenvolvimento)

DanilloMLS commented:
Também houve problema na visualização dos dados no RecyclerView, nem todos os dados aparecem

rafadescontinuado committed:
Update ClienteLogadoActivity.java

alteração de string explicita "Promoçõess"  para R.string.title_activity_promocoes

rafadescontinuado committed:
Update PromocoesActivity.java

mudanças de variáveis de acordo com padrão

DanilloMLS commented:
No últimos commit alguns erros foram relatados em PromocoesActivity, devido à renomeação dos campos.

rafadescontinuado committed:
Resoluçao bug view promoção.

DanilloMLS committed:
Merge branch 'master' into promocoes

DanilloMLS commented:
Um conflito bem simples em DefaultTest.java foi resolvido imediatamente. Algumas mudanças não puderam ser feitas por estarem interferindo no funcionamento do App.
